scripts/build-pug.js

```diff
@@ -0,0 +1,20 @@
+const { exec } = require('child_process');
+const glob = require('glob');
+const path = require('path');
+
+glob("build/chrome.*.css", function (err, files) {
+    if (err) throw err;
+
+    release = process.argv[2]; //"insights or insightsbeta"
+    chromeFileName = path.relative('build/',files[0]);
+
+    let pugvars = `{release: '${ release }', chrome:'${ chromeFileName }'}`;
+
+    exec(`pug src/pug -o build/snippets -O "${ pugvars }"`, function(err, stdout, stderr){
+        if (err) throw err;
```

Comment:
Can you add shortcutting here? `err && throw err;`
```suggestion
        err && throw err;
```

Reply:
@karelhala weird... I can't seem to do that. It gives me a `Expression expected` error

Reply:
Ah, that is strange. Nevermind, let's keep it like this for now.